exp-player/addon/components/exp-lookit-dialogue-page/component.js

```diff
@@ -518,7 +518,7 @@ export default ExpFrameBaseUnsafeComponent.extend(FullScreen, VideoRecord,  {
                  * @event clickSpeaker
                  * @param {String} imageId
                  */
-                this.sendTimeEvent('clickSpeaker', {
+                this.send('setTimeEvent', 'clickSpeaker', {
```

Comment:
Hruh. I honestly thought I'd converted this frame. Thanks for the catch, and my sincere apologies for missing it.